packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -98,6 +125,56 @@ class Send extends Component {
       token
     } = this.props;
 
+    const { estimatedTxFee, showDetails } = this.state;
+
+    const renderFee = () => {
```

Comment:
2 solutions here:
- either create a variable called `renderFee`
- or put this function as a class method.

Reason: now we're creating a new function on each render.